### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by benoror** [ecaresoft/dbMock#1]
Esta muy interesante el approach de JOOQ para probar DB connections!

Mi única pregunta por lo pronto: Faltarían agregar `assert` statements a las pruebas correcto (con `JUnit.Assert` me imagino)?

https://github.com/ecaresoft/dbMock/blob/master/src/main/java/dbmocker/MockTester.java#L27-L38

**Comment by mrojasaquino** [comment]
@benoror Claro, la idea era primero ver si era posible hacer el mockup sin necesidad de conectarse a la base de datos, y lo siguiente es crear un ejemplo de una suite o un caso de prueba.
